.github/workflows/changelog-verify.yml

```diff
@@ -34,9 +34,75 @@ jobs:
     runs-on: ubuntu-latest
     # require the first job to have ran
     needs: changedfiles
+    # Map a step output to a job output
+    outputs:
+      all: ${{ steps.validate.outputs.all }}
     # only run if there are changed files
     if: ${{needs.changedfiles.outputs.all}}
     steps:
       - uses: actions/checkout@v3
       - name: Verify changelogs
-        run: bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}"
+        id: validate
+        # Set outputs using the command.
+        run: |
+          bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}" 2>&1 | tee output.txt
+          echo "all=$(cat output.txt | jq -sR .)" >> $GITHUB_OUTPUT
+  review:
+    runs-on: ubuntu-latest
+    # require the validate job to have ran
+    needs: validate
+    if: always()
+    steps:
+      - name: Verify changelogs KO
```

Comment:
KO?

Reply:
"KO" 🥊: knock-out, usually used to define a bad outcome, contrary to the "OK"

Am I that old? Mmmmmmkay!

Reply:
Haha, that's what I had assumed but had to make sure.

I used to box when I was a kid :)